swift/ql/lib/change-notes/2023-05-23-path-injection-sinks.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added sqlite3 and SQLite.swift path injection sinks for the `swift/path-injection` query.
```

Comment:
This doesn't sound like a library change to me. So we should probably place it in `swift/ql/src/change-notes` instead. That folder doesn't exist yet for Swift, but feel free to create it.

Reply:
Following discussion, it appears to be loosely agreed that a change note belongs in the `src` or `lib` directory mostly according to who it affects (library or query users).  So I've moved this one to `src`.